oio/container_streaming/app.py

```diff
@@ -320,7 +321,7 @@ def close(self):
             self.logger.info("data not all consumed")
 
 
-class ContainerStreaming(object):
+class ContainerStreaming(RedisConn):
```

Comment:
Could you rename that? This looks more like a container backup service than like a container "streaming". And I think this does not deserve a package of its own, you should rename the file and move it into the `oio.container` package.